unittests/test_policies.py

```diff
@@ -7,8 +7,11 @@
 import itertools
 import os
 import pytest
+import signal
+import time
 import tempfile
 import unittest
+from multiprocessing import Pipe, Process
```

Comment:
```suggestion
import multiprocessing
```
And use fully qualified names later on.